### DISCUSSION TITLE
Simple Newbie Questions on States and Toggles

### DISCUSSION THREAD

**Discussion by Lusayalumino** [nguyenquyhy/Flight-Tracker-StreamDeck#339] [Q&A]
Flight Tracker is a game changer, just incredible for MSFS -- thank you!

I'm trying to learn how to use this and wanted to start with two simple items: toggle and give state for the Drone Camera (CAMERA_STATE=4) and for Drone Lock (DRONE_CAMERA_LOCKED). Man I just can't figure it out. Can anyone help?

**Comment by nguyenquyhy** [comment]
If you prefer reading documentation, take a look at our User Guide https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/blob/main/docs/USERGUIDE.md. 

You can also look at some sample profiles:
- https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/128
- https://flightsim.to/c/third-party/stream-deck/ (some here requires another software called Lorby's AAO, so you can start with those from NERIKENAUT)

**Comment by Lusayalumino** [comment]
Sorry I hadn't provided more detail... yes, I already did all that. That's how I got as far as I did. I didn't find any of the commands I'm looking for in any of the profiles I downloaded (there's ~40 that I'm looking for, but just beginning with two simple ones). It was through your user guide I learned how to use the software. Perhaps I missed something someplace? I feel like I'm missing the Legend / Key. Really appreciate any direction you can provide. Warmest regards.

**Reply by nguyenquyhy** [reply]
Does any of those work? https://docs.flightsimulator.com/html/Programming_Tools/Event_IDs/View_Camera_Events.htm
I don't see `DRONE_CAMERA_LOCKED` anywhere, so I am not too sure where you get it from.

**Comment by Lusayalumino** [comment]
I found the drone variables here: https://docs.flightsimulator.com/html/Programming_Tools/SimVars/Camera_Variables.htm#DRONE_CAMERA_LOCKED – but haven’t been able to get them to work. That’s why I’m wondering if they’re prebuilt in your software?

**Reply by nguyenquyhy** [reply]
I see. Right now you cannot set a variable directly with the plugin. It must be done through a SimConnect event (https://docs.flightsimulator.com/html/Programming_Tools/Event_IDs/View_Camera_Events.htm). Variable can be used only for Feedback (ON/OFF) or Display fields.
